src/hydration/useHydrate.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
This hook is included in the `hydration`-bundle which makes the bundle React-specific. I think this makes sense for now to keep down default payload, but that this should probably move to the React-bundle if `core` and `react` are split to separate bundles in the future to keep `hydration` framework agnostic.